lib/gcloud/bigquery.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "gcloud"
+require "gcloud/bigquery/project"
+
+##
+# Google Cloud Bigquery
```

Comment:
I had found some Java code that used "Bigquery". I originally had "BigQuery", but changed it to be more inline with the other code I was seeing. Easy enough to change back.